tool/src/main/java/org/wildfly/security/tool/FileSystemEncryptRealmCommand.java

```diff
@@ -613,7 +613,8 @@ private void createFileSystemRealm() throws Exception {
                 continue;
             }
             CredentialStore credentialStore;
-            if ((descriptor.getCredentialStore() == null) == (descriptor.getSecretKeyAlias() == null)) {
+//            check if credential-store and secret-key-alias are both specified, or both null
```

Comment:
Minor style comment. The `//` should be next to the `check`, e.g.,

```
if (...)
    // check...
...
```

Reply:
Fixed